packages/cli/src/validate-scan-arguments.ts

```diff
@@ -24,4 +24,15 @@ export function validateScanArguments(args: ScanArguments): void {
     if (args.updateBaseline === true && isEmpty(args.baselineFile)) {
         throw new Error('Option --updateBaseline requires option --baselineFile.');
     }
+
+    if (!args.crawl && (!isEmpty(args.serviceAccountName) || !isEmpty(args.serviceAccountPass))) {
+        throw new Error('Options --serviceAccountName and --serviceAccountPass are only supported with --crawl.');
+    }
+
+    if (
+        (isEmpty(args.serviceAccountName) && !isEmpty(args.serviceAccountPass)) ||
+        (!isEmpty(args.serviceAccountName) && isEmpty(args.serviceAccountPass))
+    ) {
+        throw new Error('Both --serviceAccountName and --serviceAccountPass must be provided for authenticated crawling.');
```

Comment:
do you mean `must be provided for website authentication`?

Reply:
what about `must be provided to scan authenticated pages` ?

Reply:
ok